vuepress/docs/v7.0/tutorials/create/pb/publish-simple-bundle.md

```diff
@@ -4,138 +4,146 @@ sidebarDepth: 2
 
+This tutorial describes how to create a simple Entando Bundle and deploy it into the [Entando Component Repository](../../../docs/getting-started/concepts-overview.md#entando-component-repository) (ECR). This process consists of 4 essential steps:
+
+1. Define the bundle component(s)
+2. Check the bundle artifacts into a Git repository
+3. Apply the bundle custom resource to Kubernetes
+4. Install the bundle into an application
+
+The sections below detail the steps required to generate simple examples of each Entando bundle type.
 ## Prerequisites
-* Use the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment) to verify all dependencies are installed (e.g. Java, npm, Git).
-``` sh
-ent check-env develop
-```
-* Authenticated Git credentials, an empty Git repository and an available Entando instance are required for the commands below to execute without errors.
+* Verify dependencies with the [Entando CLI](../../../docs/reference/entando-cli.md#check-the-environment): `ent check-env develop`
+* Authenticated Git credentials
+* An empty Git repository
+* A running Entando instance
 
 Publishing a bundle can be simplified by using the `ent prj` command and its publication system (pbs) convenience methods. Both the CLI and manual commands are provided.
 
 ## Create the Project Structure
-First, create a parent project directory (e.g. `example-bundle`) along with a child bundle directory. In a project generated by the [Entando Component Generator](../../../docs/create/component-gen-overview.md), the microservice and micro frontend source files live under the parent directory.
 
-``` sh
-mkdir -p example-bundle/bundle; cd example-bundle/bundle
-```
+1. Create a parent project directory (e.g. `example-bundle`) and a child `bundle` directory:
+   ``` sh
+   mkdir -p example-bundle/bundle; cd example-bundle/bundle
+   ```
+
+   In a project generated by the [Entando Component Generator](../../../docs/create/component-gen-overview.md) (ECG), the microservice and micro frontend source files are stored in the parent directory.
 ## Add a Simple Widget
 
-Create a widget directory
-``` sh
-mkdir widgets
-```
-
-Create a widget descriptor file within that directory
-``` sh
-touch widgets/example-widget.yaml
-```
-
-Populate the widget descriptor file `example-widget.yaml` with a simple definition. Make sure to retain the correct YAML indentation of 2 or 4 spaces.
-``` yaml
-code: example-widget
-titles:
-   en: Example Widget
-   it: Widget d'esempio
-group: free
-customUi: <h2>Hi from Example Widget</h2>
-```
+1. Create a widget directory:
+   ``` sh
+   mkdir widgets
+   ```
+
+2. Create a widget descriptor file within that directory:
+   ``` sh
+   touch widgets/example-widget.yaml
+   ```
+
+3. Populate the widget descriptor file `example-widget.yaml` with a simple definition. Retain the correct YAML indentation of 2 or 4 spaces.
```

Comment:
Retain the correct YAML indentation of 2 or 4 spaces. >> I know what it is saying but it feels like some context is needed. What does YAML indentation in and of itself mean? YAML indentation format/rules?